Service/AddOn/ReadmeBuilder.php

```diff
@@ -472,23 +472,49 @@ protected function generateHtml() : string
                 $block .= '</tbody></table>';
             };
 
+            $generateListFromEntity = function($entities, array $headerMap) use (&$block, &$readme)
```

Comment:
I believe `$entities` can be type-hinted as `XF\Mvc\Entity\ArrayCollection`.

Reply:
`$data['cli_commands']` is not an `ArrayCollection`, and type hinting `$entities` here will cause issues in the future if the table definition is changed.